test/input.txt

```diff
@@ -140,6 +140,47 @@
   delete 1
   list
 
+##########################################################
+# test edit person command
+##########################################################
+
```

Comment:
Add a test when there is no last active view of the list. (like there is one for view and viewall)